php/Terminus/Helpers/UpdateHelper.php

```diff
@@ -17,11 +17,13 @@ class UpdateHelper extends TerminusHelper
     public function getCurrentVersion()
     {
         $request  = new Request();
-        $url      = 'https://api.github.com/repos/pantheon-systems/terminus/releases';
-        $url     .= '?per_page=1';
-        $response = $request->request($url, ['absolute_url' => true]);
-        $release  = array_shift($response['data']);
-        $cache    = new FileCache();
+        $url = 'https://api.github.com/repos/pantheon-systems/terminus/releases/tags/0.x?per_page=1';
```

Comment:
Sure, but there will certainly remain users who will want to use 0.x for a time. Adding this now means that there will be more people who will have this change once 1.0.0 becomes the latest release.